init.lua

```diff
@@ -107,14 +107,15 @@ function irc:connect()
 			port = self.config.port,
 			password = self.config.password,
 			timeout = self.config.timeout,
+			reconnect = self.config.reconnect,
 			secure = self.config.secure
 		})
 	end)
 
 	if not good then
 		minetest.log("error", ("IRC: Connection error: %s: %s -- Reconnecting in ten minutes...")
 					:format(self.config.server, message))
-		minetest.after(600, function() self:connect() end)
+		minetest.after(self.config.reconnect, function() self:connect() end)
```

Comment:
Ahh, I gotcha. Thanks for clarifying. I've updated the pull request accordingly and appreciate the correction.